packages/asset-swapper/src/utils/quote_report_generator.ts

```diff
@@ -17,13 +18,15 @@ export interface BridgeReportSource {
     liquiditySource: Exclude<ERC20BridgeSource, ERC20BridgeSource.Native>;
     makerAmount: BigNumber;
     takerAmount: BigNumber;
+    fillData?: FillData;
```

Comment:
in what cases is `fillData` undefined for `BridgeReportSource`?

Reply:
I am not sure actually, the `DexSample` type has it as potentially undefined so I marked it like that as well. We only need the `fillData` for gas calculations with some of the integrations.